README.md

```diff
@@ -211,22 +279,7 @@ curl -sk -u broker:secret -H "X-Broker-Api-Version: 2.9" https://127.0.0.1:9293/
 cf service-access # should show all services as enabled, cf marketplace should show the same
 ```
 
-### Launch the Operators
-For running lifecycle operations, corresponding operator processes have to be started. Currently Service Fabrik Broker supports Bosh Director based and Docker Based services.
-
-Assuming that all required env variables (SETTINGS_PATH, NODE_ENV) are already set. Both bosh operator and docker operator can be launched.
-```shell
-node operators/StartBoshOperators.js #to start bosh operator
-node operators/StartDockerOperators.js #to start docker operator
```

Comment:
I think these also need to be adapted based on restructuring.

Reply:
Yes, can you please suggest a change which I can commit ?

Reply:
For now,

```
node applications/operators/StartBoshOperators.js #to start bosh operator	
node application/operators/StartDockerOperators.js #to start docker operato
```
I think I will again have change due to https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/823

Reply:
added your commit suggestions